examples/cluster/plot_kmeans_digits.py

```diff
@@ -53,14 +53,13 @@
 
 
 print(79 * '_')
```

Comment:
If it doesn't cause problems in the generated docs I can't think of any other considerations to account for.. I also updated the surrounding lines just now